nf-test.config

```diff
@@ -3,5 +3,5 @@ config {
     testsDir "tests"
     workDir ".nf-test"
     configFile "tests/nextflow.config"
-    profile "test,docker"
+    profile "docker"
```

Comment:
should the docker profile be preferred over singularity?

Reply:
it simplifies the execution in GA, no need to setup singularity... that is the only reason I left it like that